src/Component/src/Metadata/Resource/Factory/AttributesResourceClassListFactory.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Metadata\Resource\Factory;
+
+use Sylius\Resource\Metadata\AsResource;
+use Sylius\Resource\Metadata\Resource\ResourceClassList;
+use Sylius\Resource\Reflection\ClassReflection;
+
+/**
+ * Creates a resource class list from {@see AsResource} attributes.
+ *
+ * @experimental
+ */
+final class AttributesResourceClassListFactory implements ResourceClassListFactoryInterface
+{
+    /**
+     * @param array{paths: string[]} $mapping
+     */
```

Comment:
It can be in one line, similar as other doc blocks

Reply:
fixed.